spec/features/admin/admin_adds_items_spec.rb

```diff
@@ -16,14 +16,23 @@
     click_on "Edit Items"
   end
 
-  scenario 'can add new item' do
+  xscenario 'can add new item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.